### ISSUE TITLE
Permanent deletion of accounts

### ISSUE THREAD

**Issue by il-palmi** [WEEE-Open/crauto#43]

**Comment by lvps** [comment]
@il-palmi i.e. add a button for it, for admins only. Also ask for confirmation, to avoid spiacevoli incidenti

**Comment by alecello** [comment]
@lvps May i suggest something github-style where you have to type the username in correct case for it to proceed? Just to be on the extra safe side, considering the potential impact of the task

**Comment by lvps** [comment]
@alecello Yeah, sounds good! Also, having something to delete invites (not just accounts) would be useful, too
